frontend/packages/component-library/src/dropdown/simpleDropdown/SimpleDropdown.tsx

```diff
@@ -13,9 +13,12 @@ export interface SimpleDropdownProps
   extends DropdownFormFieldRelatedProps,
     Omit<SelectHTMLAttributes<HTMLSelectElement>, 'size'> {
   /** SimpleDropdown items list */
-  items?: {value: string; text: string}[];
+  items?: {value: string; text: string; disabled?: boolean}[];
```

Comment:
Nit/suggestion: create a shared `ItemsProps` interface:
 
```suggestion
export interface ItemProps {
  value: string;
  text: string;
  disabled?: boolean;
}

...
  items?: ItemProps[];
  ...
  itemGroups?: {
    label: string;
    groupItems: ItemProps[];
```